src/Miunie.Discord/CommandModules/HelpCommand.cs

```diff
@@ -0,0 +1,46 @@
+﻿// This file is part of Miunie.
+//
+//  Miunie is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  Miunie is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with Miunie. If not, see <https://www.gnu.org/licenses/>.
+
+using Discord.Commands;
+using Miunie.Core.Providers;
+using System.Threading.Tasks;
+
+namespace Miunie.Discord.CommandModules
+{
+    [Name("Help")]
+    public class HelpCommand : ModuleBase<SocketCommandContext>
+    {
+        private readonly HelpService _helpService;
+
+        public HelpCommand(CommandHandler commandHandler, ILanguageProvider lang)
+        {
+            _helpService = commandHandler.GetHelpService(lang);
+        }
+
+        [Command("help")]
+        [Summary("HELP_HELP")]
```

Comment:
I think there is no practical need for a `.resx` of our summaries. You would have to switch to and from the `.resx` file and you can still very easily just misspell it.

Not to mention the summaries are usually fairly short in nature.

We also don't and won't translate the Discord layer at all, so given the commands are for example in plain strings, I think Summaries should be, too.

Also, we should probably get the examples in as well, don't you think? 🤔

Reply:
I was kind of assuming this as well. I'll get on to reverting it.